distribution/examples/loadbalancing/3-client/membrane.sh

```
@@ -35,7 +50,7 @@ start_membrane() {
 if ! ( _test=test && _="${_test#t}" ) >/dev/null 2>&1; then
```

Shellcheck (0.10.0)

[warning] Use var=$(command) to assign output (or quote to assign string).

(SC2209)